pandas/io/formats/html.py

```diff
@@ -79,7 +79,7 @@ def row_levels(self) -> int:
         # not showing (row) index
         return 0
 
-    def _get_columns_formatted_values(self) -> ABCIndex:
+    def _get_columns_formatted_values(self) -> Iterable[Any]:
```

Comment:
Can just make this `Iterable` - if you don't subscript the generics they default to inclusive of `Any` anyway